package.json

```diff
@@ -86,6 +86,7 @@
     "jest": "^24.9.0",
     "lerna": "^3.15.0",
     "lint-staged": "^10.0.7",
+    "prettier": "^1.16.4",
```

Comment:
that's a good idea, but not i the scope of this issue, probably that's the reason of breaking build (linter) - if this is not a problem, could you introduce pretier as a separated PR? but if you are able to fix all of build issues quickly - it's fine

Reply:
@andrzejewsky I've fixed the eslint tracked errors.

Reply:
Well... Actually I removed it. Will create separate PR for that. 🙈